python/cudf/cudf/utils/ioutils.py

```diff
@@ -1215,6 +1215,8 @@
 delimiter : string, default None
     The delimiter that should be used for splitting text chunks into
     separate cudf column rows. The delimiter may be one or more characters.
+strip_delimiters : boolean, default False
+    If True, removes delimiters that are normally kept at the end of strings.
```

Comment:
I don't think this docstring is detailed enough to explain the behavior. It's hard to put a real example in the parameters section, but maybe we can provide more info below?

Should this be named `strip_trailing_delimiters` to clarify that it acts at the end of the input line? (or entire input file? unclear) Python's default is to strip both sides:
```python
assert "asdfa".strip("a") == "sdf"
```

Reply:
Short docstrings are usually okay when we are matching a pandas API, because users have some expectations for how the pandas API might work. Since `read_text` is not aligned with a pandas API, detailed descriptions and examples are important for setting user expectations.

Reply:
Agreed, we also don't mention anywhere that the delimiters are kept at the end. I'll extend it a bit

Reply:
WDYT?
```suggestion
    Unlike the `str.split()` function, `read_text` preserves the delimiter
    at the end of a field in output by default, meaning `a;b;c` will turn into
    `['a;','b;','c']` when using `;` as a delimiter.
    Setting this option to `True` will strip these trailing delimiters,
    leaving only the contents between delimiters in the resulting column:
    `['a','b','c']`
```

Reply:
That is very helpful info! I’m happy with that.